src/utils.rs

```diff
@@ -22,7 +22,8 @@
 #![allow(dead_code)] // This module acts more like a library, so not yet used is ok.
 
 use log::debug;
-use std::env;
+use nix::unistd::{Uid, User};
+use std::ffi::OsString;
```

Comment:
All other use statements in this file `use`s the module (exception: log macros and traits) and `use`s the types only in the function body.

Reply:
Okay,  Noted.
Should this convention be followed througout the project?

Reply:
Only in files where "other"-function/types/traits/… are. Currently `macros/src/lib.rs` and `src/utils.rs`.